servers/rendering/renderer_canvas_cull.cpp

```diff
@@ -721,8 +721,10 @@ void RendererCanvasCull::canvas_item_add_circle(RID p_item, const Point2 &p_pos,
 	static const int circle_points = 64;
 
 	points.resize(circle_points);
+	real_t circle_point_step = Math_TAU / circle_points;
```

Comment:
Would be nice put `const` here.

Reply:
Done for all the `_step` variables.

Now that I think about it, there are probably a ton of places around the codebase where `const` can be used. There is probably some tool we can use to look for places to add it.